modules/installation-aws-edge-compute-pools-examples.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+// * installing/installing-aws-localzone.adoc (Installing a cluster on AWS with worker nodes on AWS Local Zones)
+// * installing/installing-aws-wavelength-zone.adoc (Installing a cluster on AWS with worker nodes on AWS Wavelength Zones) 
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-aws-edge-compute-pools-examples_{context}"]
+= Examples of installation configuration files with edge compute pools
+
+The following examples show `install-config.yaml` files that contain an edge machine pool configuration.
+
+.Configuration that uses an edge pool with a custom instance type
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+metadata:
+  name: ipi-edgezone
+compute:
+- name: edge
+  platform:
+    aws:
+      type: m5.4xlarge
+platform:
+  aws:
+    region: us-west-2
+pullSecret: '{"auths": ...}'
+sshKey: ssh-ed25519 AAAA...
+----
+
+// Q: Duplicated. Why not reference the "Features" AWS documentation here? or CLI to check the offerings?
+Instance types differ between locations. To verify availability in the {zone-type} in which the cluster runs, see the AWS documentation.
+
+.Configuration that uses an edge pool with a custom Amazon Elastic Block Store (EBS) type
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: devcluster.openshift.com
+metadata:
+  name: ipi-edgezone
+compute:
+- name: edge
+  platform:
+    aws:
+      rootVolume:
+        type: gp3
+        size: 120
```

Comment:
Probably it would be nice to improve this example too, it is linked with supported zones in the features page[1]. WDYT @yunjiang29 ?

```suggestion
    aws:
      zones:
      - us-west-2-lax-1a
      - us-west-2-lax-1b
      - us-west-2-phx-2a
      rootVolume:
        type: gp3
        size: 120
```

[1] https://aws.amazon.com/about-aws/global-infrastructure/localzones/features/

Reply:
Great.

Reply:
Thanks, Marco. Updated.